interfaces/hotplug/spec.go

```diff
@@ -34,60 +34,52 @@ type Definer interface {
 
 // SlotSpec is a definition of the slot to create in response to udev event.
 type SlotSpec struct {
```

Comment:
Why not call it `RequestedSlotSpec` or `DesiredSlotSpec`?

Reply:
I like `RequestedSlotSpec`, thanks, changed.